source/game.gamescriptexpression.bmx

```diff
@@ -39,7 +39,8 @@ GameScriptExpression.RegisterFunctionHandler( "worldtime", SEFN_WorldTime, 1, 1)
 Function SEFN_WorldTime:SToken(params:STokenGroup Var, context:SScriptExpressionContext)
 	Local command:String = params.GetToken(0).GetValueText()
 	Local subCommand:String = params.GetToken(1).value 'MUST be a string
-	
+
+	'TODO formatted date, weekdayname?
```

Comment:
weekdayname -> wenn, dann aber wohl nur auf "englisch" (alles andere ergaebe keinen Sinn).
formatted date -> wozu? Damit man "y/m/d"-Vergleiche machen kann?

Reply:
Wochentag: gen_settings enthält ja für alle Wochentage die Kurz- und Langvarianten. Das könnte (langfristig) für Texte interessant sein.

Beim Datum habe ich an tatsächliche Datumsangaben in Texten gedacht (z.B. Geburtstag von x), wobei Person aktuell diese Information noch gar nicht zur Verfügung stellt.

Reply:
Ja aber da musst Du Dich am Ende noch mit dem Thema "Datums-Format" herumschlagen.

Reply:
Im Spiel gibt es ja schon ausgewertete Formate. Wenn man die dokumentiert, könnte man sie direkt verwenden.